src/components/Whiteboard/ElementBehaviors/Resizable/ResizeHandle.tsx

```diff
@@ -21,9 +21,11 @@ import { ResizeHandlePosition } from './types';
 
 export function calculateResizeHandlePosition(
   position: ResizeHandlePosition,
-  containerWidth: number,
-  containerHeight: number,
+  containerWidth: number = 0,
+  containerHeight: number = 0,
   scale: number,
+  positionX: number = 0,
+  positionY: number = 0,
```

Comment:
Would be good to have this explicit in API that for `start`and `end` (for lines) we use `positionX` and `positionY` only and `top`, `topRight` (for others) we use `containerWidth` and `containerHeight`. Otherwise looks not clear what should be the expected valid combinations or these parameters with these many default values without checking other code.